server/db/models/Product.js

```diff
@@ -1,12 +1,8 @@
-const { STRING } = require('sequelize')
+const { DataTypes } = require('sequelize')
 const db = require('../db')
 
 
-
-module.exports = db.define('Product', {
-
-    name: {
-        type: STRING
-    }  
-
-})
+module.exports = db.define("product", {
```

Comment:
Hi Kevin,I understand you will need this module to complete the code..Can I push it first?Or how would we resolve the git conflict??